File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/scenario/ScenarioCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.scenario;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class ScenarioCreatorHub extends CreatorHub {
    public Scenario createScenario(String scenarioID, AbstractDataProvider dataProvider) throws Exception {
        return (Scenario) this.create(scenarioID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/scenario/ScenarioCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.scenario

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class ScenarioCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): Scenario {
        return super.create(itemID, dataProvider) as Scenario
    }
}